arex-storage-web-api/src/main/java/com/arextest/storage/repository/RepositoryProviderFactory.java

```diff
@@ -56,6 +56,10 @@ public MockCategoryType findCategory(String categoryName) {
       return MockCategoryType.COVERAGE;
     }
 
+    if (MockCategoryType.RECORDING_SCENE.getName().equals(categoryName)) {
+      return MockCategoryType.RECORDING_SCENE;
```

Comment:
why not get from `categoryTypeMap.get(categoryName);` ?

Reply:
There is only the default category type in categoryTypeMap